README.md

```diff
@@ -181,6 +182,34 @@ const b = asUnit('1rem') // returns '1rem'
 const c = asUnit(null) // Throws a TypeError
 ```
 
+`asMaybe` creates a cleaner that doesn't throw on an invalid type. It tries the cleaner, and if that throws an exception, it will return undefined instead:
+
+```typescript
+// Makes a Cleaner<string | undefined>:
+const asMaybeString = asMaybe(asString)
+
+const a = asUnit('Valid string') // returns 'Valid string'
+const b = asUnit(23) // returns undefined
+const c = asUnit(null) // returns undefined
```

Comment:
asUnit?

Reply:
Ah, my copy pasting skills are not as good as they once were. 😆